NOTES.md

```diff
@@ -1,3 +1,107 @@
+# Release 2.1.0-beta (Forbidden Planet)
+
+-- issues up-to-date with the 2.1 milestone as of 2015-07-06 23:00 UTC
+-- contributors " " next as of 3dac994 and docs:master as of 0b661c4
+
+Release highlight:
+
+* Automatic failover
+* More flexible administration for servers and tables
+
+Read the [blog post][2.1-beta] for more details.
+
+[2.1-beta]: http://rethinkdbblog.com/blog/2.1-beta/
+
+## Compatibility ##
+
+This beta release does not include automatic migration of data
+directories from older versions of RethinkDB. However the final 2.1.0
+version will include that feature for versions of RethinkDB down to
+1.14.0. Upgrading directly from RethinkDB 1.13 will no longer be
```

Comment:
Rather than say "will no longer be supported", can we say "will require manually upgrading using `rethinkdb export`"?